CesiumRasterOverlays/src/DebugText.h

```diff
@@ -0,0 +1,242 @@
+#include <CesiumGltf/ImageAsset.h>
+
+#include <algorithm>
+#include <cstdint>
+
+namespace CesiumRasterOverlays {
+
+namespace {
```

Comment:
It's not great to put an anonymous namespace in a header file, because it means every cpp that includes this will get its own copy. The best approach is to add a `DebugText.cpp` and move as much as possible (pretty much all of it) into there.